packages/web/pages/apps/index.tsx

```diff
@@ -80,6 +86,25 @@ export const AppStore: React.FC<AppStoreProps> = ({ apps }) => {
 
   const iterableData = searchValue ? fuzzySearchResults : nonFeaturedApps;
 
+  const getSize = useCallback(() => {
```

Comment:
This handles making sure the search bar isn't too big

Reply:
https://github.com/osmosis-labs/osmosis-frontend/blob/2af65d6456e7baca144f70bc2562309e4ac7616b/packages/web/components/types.ts#L51

We have these breakpoints, which align with our tailwind breakpoints. Using those allows all breakpoints to occur at the same point (from CSS and JS), instead of having them confusingly occur at different points given these hardcoded values.

Generally across this entire review, the Apps page is fully isolated so really anything works. But in case other devs come here for reference, we should follow established conventions.